{"name": "Проверка неуспешной активации пользователя, при отсутствии токена", "status": "passed", "steps": [{"name": "Создание нового пользователя", "status": "passed", "attachments": [{"name": "request_body", "source": "333c0c55-8bdf-4b1a-9cc0-e8966d0701af-attachment.json", "type": "application/json"}, {"name": "curl", "source": "1d88d4aa-ddb2-47d4-b91e-e0438ca7b635-attachment.txt", "type": "text/plain"}, {"name": "response_body", "source": "9b026d55-2a71-4b9d-b8e6-32bbdd2e7484-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "login", "value": "'aanastya02_02_2025_15_19_50+5404'"}, {"name": "password", "value": "'123456789'"}, {"name": "email", "value": "'aanastya02_02_2025_15_19_50+5404@mail.ru'"}], "start": 1738495190004, "stop": 1738495190119}, {"name": "Активация пользователя", "status": "broken", "statusDetails": {"message": "requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: http://5.63.153.31:5051/v1/account/None\n", "trace": "  File \"C:\\Users\\user\\PycharmProjects\\dm_api_tests\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\user\\PycharmProjects\\dm_api_tests\\helpers\\account_helper.py\", line 63, in activation_user\n    response = self.dm_account_api.account_api.put_v1_account_token(token=token,\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\user\\PycharmProjects\\dm_api_tests\\dm_api_account\\apis\\account_api.py\", line 69, in put_v1_account_token\n    response = self.put(path=f'/v1/account/{token}')\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\user\\PycharmProjects\\dm_api_tests\\restclient\\client.py\", line 29, in put\n    return self._send_request(method='PUT', path=path, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\user\\PycharmProjects\\dm_api_tests\\restclient\\utilies.py\", line 16, in wrapper\n    response = fn(*args, **kwargs)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\user\\PycharmProjects\\dm_api_tests\\restclient\\client.py\", line 66, in _send_request\n    rest_response.raise_for_status()\n  File \"C:\\Users\\user\\PycharmProjects\\dm_api_tests\\venv\\Lib\\site-packages\\requests\\models.py\", line 1024, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n"}, "parameters": [{"name": "token", "value": "None"}, {"name": "validate_response", "value": "False"}], "start": 1738495190119, "stop": 1738495190153}], "attachments": [{"name": "stdout", "source": "0a8fa98f-8447-4725-827b-c1273c1215a7-attachment.txt", "type": "text/plain"}], "start": 1738495190004, "stop": 1738495190154, "uuid": "467a0452-1408-4498-913f-76feae23e465", "historyId": "3d88eb8a3bd91fdf4b393883814e8a11", "testCaseId": "3d88eb8a3bd91fdf4b393883814e8a11", "fullName": "tests.test_put_v1_account_token.TestActivationUser#test_unsuccessful_activation_user", "labels": [{"name": "suite", "value": "Тесты на проверку метода post_v1_account_token"}, {"name": "subSuite", "value": "Проверка активации пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "host", "value": "WIN-RAUHT8ELOL4"}, {"name": "thread", "value": "20956-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_put_v1_account_token"}]}